evap/evaluation/templates/base.html

```diff
@@ -37,20 +37,23 @@
 {% endblock %}
 
 {% get_current_language as LANGUAGE_CODE %}
-{% cache 3600 navbar request.session.session_key LANGUAGE_CODE %}
-    {% include_navbar user LANGUAGE_CODE %}
-{% endcache %}
-
+<div class="fixed-top d-print-none">
+    {% cache 3600 navbar request.session.session_key LANGUAGE_CODE %}
+        {% include_navbar user LANGUAGE_CODE %}
+    {% endcache %}
+    {% if user.is_staff or user.is_grade_publisher %}
```

Comment:
reviewers don't get the breadcrumb, is that still correct?

Reply:
this is outdated :)